config/logger.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const winston = require('winston')
+
+module.exports.setupLogger = function () {
+  return winston.createLogger({
+    level: process.env.NODE_ENV !== 'production' ? winston.level.debug : winston.level.info,
```

Comment:
This is a great start, as it gets more complex it may be useful to explicitly define separate production and development configurations and have the only branch be which one is used. 

Something like: 
```js
const developmentConfig = {} //...
const productionConfig = {} //... 
```